kinto_http/session.py

```diff
@@ -59,6 +59,10 @@ def __init__(
         self.timeout = timeout
         self.headers = headers or {}
 
+        # Set the default User-Agent if not already defined in the headers.
+        if "User-Agent" not in self.headers:
+           self.headers["User-Agent"] = USER_AGENT
```

Comment:
I believe this can be written  `self.headers.set_default("User-Agent", USER_AGENT)`

And it shouldn't very different from what we had before `kwargs["headers"] = {"User-Agent": USER_AGENT, **self.headers, **overridden_headers}`, where `self.headers` overrides the default user agent.

Reply:
>> I believe this can be written self.headers.set_default("User-Agent", USER_AGENT)
Good suggestion, I will make incorporate this feedback.

>> And it shouldn't very different from what we had before kwargs["headers"] = {"User-Agent": USER_AGENT, **self.headers, **overridden_headers}, where self.headers overrides the default user agent.
Functionally, it's the same thing. The only reason I made this change was just to ensure that session.headers has "User-Agent" entry so that I can manipulate it from Client's update_record() without additional checking whether "User-Agent" entry exists or not in session.headers. Let me hear your thoughts on this. Should I make this change or keep it the same as before? kwargs["headers"] = {"User-Agent": USER_AGENT, **self.headers, **overridden_headers}